src/storage/txn/scheduler.rs

```diff
@@ -74,11 +74,11 @@ pub const RESOLVE_LOCK_BATCH_SIZE: usize = 256;
 
 /// After the GC scan of a key, output a message to the log if there are at least this many
 /// versions of the key.
-const GC_LOG_FOUND_VERSION_THRESHOLD: usize = 30;
+pub const GC_LOG_FOUND_VERSION_THRESHOLD: usize = 30;
```

Comment:
Does 30 big enough? I think this will print too many logs.

Reply:
it is chosen for 10 minutes gc interval.

Reply:
Actually when a cluster is big enough, a round GC may takes several hours to finish.